package-release.sh

```diff
@@ -27,68 +29,68 @@ opt_winelib=0
 
 crossfile="build-win"
 
-while [ $# -gt 0 ]; do
-  case "$1" in
-  "--no-package")
-    opt_nopackage=1
-    ;;
-  "--dev-build")
-    opt_nopackage=1
-    opt_devbuild=1
-    ;;
-  "--winelib")
-    opt_winelib=1
-    crossfile="build-wine"
-    ;;
-  *)
-    echo "Unrecognized option: $1" >&2
-    exit 1
+for arg; do
```

Comment:
The while loop is perfectly portable and potentially more robust. I think its also more portable to do `for arg do` instead of `for arg; do`.